docs/modules/servers/pages/basic/help.adoc

```diff
@@ -1,5 +1,54 @@
 = James Basic Server &mdash; Where to get help
 :navtitle: Help
 
-(TODO)
+////
 
+This information is based on the email discussion from here:
+  --> https://www.mail-archive.com/server-dev@james.apache.org/msg66420.html
```

Comment:
This entire page spirit do not fit the Apache contribution guidelines IMO.

Reply:
I agree! I was very unhappy with how this discussion went, and I think we ought to take a second look.

Reply:
I have started a discussion on the mailing list to hopefully resolve these issues. I will resolve this comment for now. Thanks!